mobile/src/components/MosipAuth/index.js

```diff
@@ -15,7 +15,8 @@ export const MosipAuth = () => {
     const [error, setError] = useState(null);
     const {goNext} = useWalkInEnrollment();
     const [maskedEmail, setMaskedEmail] = useState('');
-    const [maskedMobile, setMaskedMobile] = useState('')
+    const [maskedMobile, setMaskedMobile] = useState('');
+    const [isChecked, setIsChecked] = useState(false);
```

Comment:
rename to `consentProvided, setConsentProvided`

Reply:
ok